reactivesocket-cpp/src/Frame.cpp

```diff
@@ -30,6 +29,10 @@ std::unique_ptr<folly::IOBuf> FrameBufferAllocator::allocateBuffer(
   return folly::IOBuf::createCombined(size);
 }
 
+std::unique_ptr<folly::IOBufQueue> FrameBufferAllocator::allocateQueue() {
```

Comment:
I would suggest keeping this function only if it provides some value. I understand the argument about consistency, but for the reader, its easier to see a local variable on the stack than calling into a function.
You are also adding another allocation for every frame because you are returning unique_ptr<IOBufQueue> which is unnecessary.
Having the IOBufQueue as a local variable on the stack would be enough, I think.